src/buildtools/fslex/fslex.fs

```diff
@@ -13,7 +13,7 @@ open System.Collections.Generic
 open System.IO 
 
 //------------------------------------------------------------------
-// This code is duplicated from FSharp.Compiler.UnicodeLexing
+// This code is duplicated from Microsoft.FSharp.Compiler.UnicodeLexing
```

Comment:
missed a namespace change here

Reply:
This is just copying over the canonical source code from FsLexYacc, we'd need to make the change there